lib/jekyll-readme-index/generator.rb

```diff
@@ -26,15 +28,38 @@ def readme
     end
 
     def index?
-      (site.pages + site.static_files).any? { |file| file.url =~ INDEX_REGEX }
+      relative_index?(readme)
     end
 
     def page
+      relative_page(readme)
```

Comment:
What's going on here if the `page` method isn't getting passed a `readme` object?